node/src/components/rpc_server/rpcs/info.rs

```diff
@@ -60,11 +60,11 @@ impl DocExample for GetDeployParams {
 /// The execution result of a single deploy.
 #[derive(Serialize, Deserialize, Debug, JsonSchema)]
 #[serde(deny_unknown_fields)]
-pub struct JsonExecutionResult {
+pub struct JsonBlockHashAndExecutionResult {
```

Comment:
If we rename the _new_ `JsonExecutionResult` back to `ExecutionResult` then this can remain being `JsonExecutionResult`. In this case it does make sense - it's a type for JSON API only.

Reply:
I see that you reverted the rename of `ExecutionResult` @dwerner , can we also revert _this_ renaming?